roles/template_service_broker/tasks/install.yml

```diff
@@ -31,7 +31,10 @@
 
 - name: Apply template file
   shell: >
-    oc process -f "{{ mktemp.stdout }}/{{ __tsb_template_file }}" --param API_SERVER_CONFIG="{{ lookup('file', __tsb_files_location ~ '/' ~ __tsb_config_file) }}" --param IMAGE="{{ template_service_broker_prefix }}{{ template_service_broker_image_name }}:{{ template_service_broker_version }}" | kubectl apply -f -
+    oc process -f "{{ mktemp.stdout }}/{{ __tsb_template_file }}"
+    --param API_SERVER_CONFIG="{{ lookup('template', 'apiserver-config.yaml.j2') }}"
```

Comment:
"templates"?  (the dir name is templates, anyway....i dunno what "lookup" does exactly)

Reply:
(and does lookup process the jinja(?) before feeding it in as a template parameter value?)

Reply:
'template' is correct, it specifies the lookup behaviour, not the location.  And yes, with this behaviour set, lookup() returns the template file contents, processed.

Reply:
what is lookup's search behavior?  I can easily see us having more than one "apiservier-config.yaml.j2" file in the future....

Reply:
as I understand it, in the templates/ directory of the current role

Reply:
ok

Reply:
We should not have that file within the role. Ultimately we will be syncing that file from `origin` to that directory so that it is role independent. Which means that the change to the template you are suggesting below belongs in the `origin` repo instead.